src/test/java/guitests/BoardAutoCreatorTest.java

```diff
@@ -104,15 +104,23 @@ public void sampleBoardAutoCreationTest() {
 
         waitUntilNodeAppears(SAMPLE_BOARD_DIALOG);
         click("OK");
+        verifyBoard(panelControl, BoardAutoCreator.getSamplePanelDetails());
+    }
 
-        assertEquals(panelControl.getPanelCount(), SAMPLE_PANEL_NAMES.size());
-        assertEquals(panelControl.getCurrentlySelectedPanel(), Optional.of(0));
-        assertEquals(panelControl.getNumberOfSavedBoards(), 1);
-
-        List<PanelInfo> panelInfos = panelControl.getCurrentPanelInfos();
-        for (int i = 0; i < SAMPLE_PANEL_NAMES.size(); i++){
-            assertEquals(panelInfos.get(i).getPanelFilter(), SAMPLE_PANEL_FILTERS.get(i));
-            assertEquals(panelInfos.get(i).getPanelName(), SAMPLE_PANEL_NAMES.get(i));
+    /**
+     * Tests panelControl properties associated with panels.
+     * PanelDetails obtained from Boards are tested against current Panel infos as well.
```

Comment:
I didn't think writing a proper comment for a method takes this many iterations :-p
```
Confirms the currently displayed board consists the set of panels specified in panelDetails
```
May need to rename the method to match the description `verifyCurrentBoard`